src/AppBundle/Command/ImportSaleCommand.php

```diff
@@ -88,7 +100,7 @@ private function saveSales($jsonSales, $existingInvoices)
                 $sale->setFromJSON($jsonSale);
                 $this->em->persist($sale);
 
-                if (true == $this->getContainer()->getParameter('new_sale_notification')) {
```

Comment:
Just put the whole check here:

`if (true == $input->getOption('new-sale-notification'))`